about/policy/sgid/index.md

```diff
@@ -10,60 +10,88 @@ categories: []
 
-## Adding New Tables
+## Adding New Datasets to the SGID Geodatabase
 {: .text-left }
 
-When adding new data to the SGID, the steps outlined in the [Porter](https://github.com/agrc/porter) project will be followed.
+We strive to achieve and maintain a high standard of quality for the SGID. We use the following policies and procedures to guide this effort.
 
-Additional, related info:
+### Qualifying Datasets
+{: .text-left }
+
+Datasets should meet all the following criteria:
+
+- The dataset should cover a **state-wide** geographic extent, be part of a state-wide project or initiative (such as roads, address points, and parcels), or be relevant to broader projects or mandates that have a state-wide interest (e.g. geologic hazard mapping or endangered species mapping).
+- The dataset should be useful to **multiple agencies or organizations'** daily activities or decision-making. Datasets primarily used internally by a single agency aren't great candidates for the SGID.
+- The entity providing the dataset should be the **authoritative** or original source (for example, a county's parcels must come from the County government).
+- The entity providing the dataset does not have the **ability** to publicly share data (such as an existing ArcGIS Online organization). An entity's data storage costs or server capacity may not be sufficient justification for adding data to the SGID.
+
+Agencies and organizations can still share datasets that don't meet the guidelines above with our [Open Data site](https://opendata.gis.utah.gov/). This is a great way to share data mainly intended for the general public with a wide audience. Please see our [Open Data sharing guidelines]({%link gis-data-sharing/open-data/index.md %}) for more information.
+
+We will periodically review the SGID for datasets that no longer meet these criteria. Specifically, we may consider removing datasets that see little to no usage for a prolonged period of time.
+
+### Dataset Introduction Process
+{: .text-left }
 
-A corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1) will be added, and will contain at a minimum:
+We will use the [Porter](https://github.com/agrc/porter) project to guide and track the steps needed to add or make breaking changes (remove, schema change, etc.) to products contained in the SGID. A brief overview of these steps is included below.
+
+We will add a corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1). This will contain at a minimum:
 
 - The authoritative access from location
 - The table name
 - A description
 - The data steward
 - The steward contact name and email
-- A link the the data page on the AGRC website
+- A link on the data page on the AGRC website
```

Comment:
```suggestion
- A link to the data page on the AGRC website
```
Since we're talking about the spreadsheet here, this should be a link 'to' the data page of the website.

We also might need to more clearly label the columns of the spreadsheet that pertain to these bullets (Source --> Steward, Website URL, Endpoint, etc.)

Reply:
Lets coordinate any spreadsheet column name changes as quite a few automation tools depend on those.